### PR TITLE
release(hanoi): rfid-llrp 1.0.0

### PR DESCRIPTION
**Pull request by bill-mahoney**
Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

### CODE REVIEW

File: release/app-rfid-llrp-inventory.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: 'app-rfid-llrp-inventory'
+version: '1.0.0'
+releaseName: 'ireland'
```
Comment (lenny-intel) [inline release/app-rfid-llrp-inventory.yaml:4]: Should be `hanoi` since it is a 1.x release

File: release/device-rfid-llrp-go.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: 'device-rfid-llrp-go'
+version: '1.0.0'
+releaseName: 'ireland'
```
Comment (lenny-intel) [inline release/device-rfid-llrp-go.yaml:4]: Should be `hanoi` since it is a 1.x release

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: release/device-rfid-llrp-go.yaml (hunk 2 above)
Comment (bill-mahoney) [reply release/device-rfid-llrp-go.yaml]: Not sure we use `releaseName` for anything actually but I changed it to be consistent.

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**